### ISSUE TITLE
Remove phenotypes from Uniprot-OMIM linkset

### ISSUE THREAD

**Issue by stain** [openphacts/IdentityMappingService#12]
> The linkset seems to contain both genes and phenotypes! I don't think mappings of proteins to phenotypes belong to the IMS.

Linkset http://data.openphacts.org/dev/ims/linksets/uniprot/uniprot_omim.ttl.gz

**Comment by Chris-Evelo** [comment]
Agree, it doesn't belong in the IMS. But it does belong in the cache. It might be there already from the OMIM data. This could then be comparable to the ENSEMBL -> UniProt, UniProt -> ENSEML issues only for the cache and not for the IMS. It calls for an approach where we may have to choose the preferred resource for every relationship between two resources.
